packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/components/Billing/FullfillmentSummary.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react';
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { CreateReceivablesFormProps } from '@/components/receivables/InvoiceDetails/CreateReceivable/validation';
+import { useMoniteContext } from '@/core/context/MoniteContext';
+import { useRootElements } from '@/core/context/RootElementsProvider';
+import { useDateTimeFormat } from '@/utils';
+import { t } from '@lingui/macro';
+import { useLingui } from '@lingui/react';
+import { LockOutlined } from '@mui/icons-material';
+import { FormControlLabel, Checkbox, Box, Typography } from '@mui/material';
+import { DatePicker } from '@mui/x-date-pickers';
+
+import { PaymentSection } from '../../PaymentSection';
+import { SectionGeneralProps } from '../../Section.types';
+
+export const FullfillmentSummary = ({ disabled }: SectionGeneralProps) => {
+  const { i18n } = useLingui();
+  const { control, resetField, setValue, watch } =
+    useFormContext<CreateReceivablesFormProps>();
+
+  const { api } = useMoniteContext();
+
+  const { data: paymentTerms, isLoading: isPaymentTermsLoading } =
+    api.paymentTerms.getPaymentTerms.useQuery();
+
+  const { root } = useRootElements();
+  const dateTimeFormat = useDateTimeFormat();
+
+  const dateTime = i18n.date(new Date(), dateTimeFormat);
+
+  /** Describes if `Same as invoice date` checkbox is checked */
```

Comment:
Do we really need this comment? I think code is written in a good way that it is self-explanatory